bootstrap/main.go

```diff
@@ -132,14 +133,14 @@ func exitNoCapture(msg string, args ...interface{}) {
 	os.Exit(1)
 }
 
-func capture(msg string, args ...interface{}) {
+func capture(skipFrames int, msg string, args ...interface{}) {
```

Comment:
The args here are only slightly different to Capture; can we reorder them for consistency?

Reply:
The issue here is that msg, a format string, is usually just before the variadic parameters. we could put skip first everywhere but if feels a it odd.

Reply:
Ah okay 👍